optimizer_wrapper.rb

```diff
@@ -679,31 +694,38 @@ def self.compute_result_total_dimensions_and_round_route_stats(result)
   end
 
   def self.compute_route_waiting_times(route)
-    seen = 1
-    previous_end =
-      if route[:activities].first[:type] == 'depot'
-        route[:activities].first[:begin_time]
-      else
-        route[:activities].first[:end_time]
-      end
+    setup_types = %w[service pickup delivery]
```

Comment:
At the end of the day what we want to do is calculating the waiting_times given the arrival, departure, travel times of activities. So I read the function with that in mind but it is not easy to comprehend. Is there a way to simplify it?

Reply:
Sure, the draft was created in a early stage. I was searching a way to launch the tests suite with optimizer-ortools dev branch

Reply:
Well having a feedback at the early stage is always good :)